hack/common.sh

```diff
@@ -356,17 +367,26 @@ function os::build::export_targets() {
   done
 
   if [[ ${#targets[@]} -eq 0 ]]; then
-    targets=("${OS_ALL_TARGETS[@]}")
+    if [[ "${platform}" == linux/amd64 ]]; then
+      targets=("${OS_ALL_TARGETS[@]}" "${OS_SDN_COMPILE_TARGETS_LINUX_AMD64[@]}")
```

Comment:
This is wrong - the abstraction before was that you didn't have to know this info in here.  If you want to build OS_SDN_COMPILE_TARGETS, pass them in to this method.